elements/campaign-brand-messaging/campaign-brand-messaging.js

```diff
@@ -0,0 +1,46 @@
+import {
+  registerElement,
+  BulbsHTMLElement,
+} from 'bulbs-elements/register';
+import { filterBadResponse, getResponseJSON } from 'bulbs-elements/util';
+import invariant from 'invariant';
+
+class CampaignProductShot extends BulbsHTMLElement {
+
+  handleRequestSuccess (data) {
+    if (!data.brand_messaging) {
+      console.log('Brand messaging not set on this campaign');
+      return;
+    }
+
+    this.innerHTML =
+      `<div class='campaign-brand-messaging'>
+        <span>${data.brand_messaging}</span>
+      </div>`;
+  }
+
+  handleRequestError (error) {
+    console.log(error);
+  }
+
+  attachedCallback () {
+    invariant(!!this.hasAttribute('src'), 'campaign-brand-messaging component requires a src');
+    fetch(this.getAttribute('src'))
+      .then(filterBadResponse)
+      .then(getResponseJSON)
+      .then(this.handleRequestSuccess.bind(this))
+      .catch(this.handleRequestError);
+  }
+
+  attributeChangedCallback (name, previousValue, value) {
+    if (document.body.contains(this)) {
+      if (name === 'src' && previousValue !== value) {
+        this.attachedCallback();
+      }
+    }
+  }
+}
+
+registerElement('campaign-brand-messaging', CampaignProductShot);
+
+export default CampaignProductShot;
```

Comment:
I think this is an issue of the generator, but I don't see any reason not to inline the `export default` with the class definition:

```
export default class CampaignProductShot extends BulbsHTMLElement
```

If you change this, we should also update the generator: https://github.com/theonion/bulbs-elements/blob/master/generator-bulbs-elements/generators/dom-element/templates/%7B%7B%20elementPathName%20%7D%7D.js#L7